src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ArrivalEntity.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import java.time.LocalDate
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "arrival")
+data class ArrivalEntity(
+  @Id
+  var id: UUID,
```

Comment:
Heavy use of `var` instead of `val` on entities.  Ideally would go for immutable objects where possible or mutable only where needed.  Obviously I don't know how these are used.

Reply:
Have ammended 👍